common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/storage/BlockConnectionStorage.java

```diff
@@ -17,18 +17,24 @@
  */
 package com.viaversion.viaversion.protocols.protocol1_13to1_12_2.storage;
 
+import com.google.common.collect.EvictingQueue;
 import com.viaversion.viaversion.api.Via;
 import com.viaversion.viaversion.api.connection.StorableObject;
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Queue;
+
+import com.viaversion.viaversion.api.minecraft.Position;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public class BlockConnectionStorage implements StorableObject {
     private static Constructor<?> fastUtilLongObjectHashMap;
 
     private final Map<Long, SectionData> blockStorage = createLongObjectMap();
+    @SuppressWarnings("UnstableApiUsage")
+    private final Queue<Position> modified = EvictingQueue.create(5);
```

Comment:
This should be larger as players might place/break more than 5 blocks per whatever the ping to the server is seconds

Reply:
I'll be keeping it at 5, and increasing the distance to <= 2 (vs < 2). Even if you have ~250ms and are placing blocks at ~20 blocks per second, this has you covered, for anything past that, we'll just hope you aren't doing it near connected blocks and being cancelled by the server: it's just too specific of a case to warrant increasing this 5, given the potential performance implication.

Increasing the distance also solves another bug (placing a door, if the top-half connects something) because you place against the floor, but the top-half that needs to be removed is 2 blocks away from the clicked one, so pre-change it'd be broken.